packages/-ember-data/package.json

```diff
@@ -11,14 +11,14 @@
     "build": "ember build",
     "build:production": "ember build --environment=production",
     "start": "ember server",
-    "docs": "ember ember-cli-yuidoc",
+    "docs": "mkdir -p dist && ember ember-cli-yuidoc",
```

Comment:
Does this work correctly if you already have stuff in dist?

Reply:
yes

Reply:
I meant more from there being existing code inside the folder, vs just the folder itself